API_Guidelines.md

```diff
@@ -97,6 +97,31 @@ Payload
 }
 ```
 
+## POST
+
+### Save Method
+
+`Ember.save` will make a POST request *only* if the record is *new*.  
```

Comment:
the method is technically `save()`. this is good info though, put it at the beginning of POST